src/signals/incident/components/form/ContainerSelect/ContainerSelect.tsx

```diff
@@ -1,33 +1,47 @@
+import type { FunctionComponent } from 'react';
 import React, { useCallback, useState } from 'react';
-import PropTypes from 'prop-types';
 import { ContainerSelectProvider } from './ContainerSelectContext';
 import Intro from './Intro';
 import Selector from './Selector';
 import Summary from './Summary';
-
-const ContainerSelect = ({ handler, meta, parent }) => {
-  const { value } = handler();
+import type { ClickEventHandler, Item, Meta } from './types';
+import type { Incident } from 'types/incident';
+import type { LatLngExpression } from 'leaflet';
+
+export interface ContainerSelectProps {
+  handler: () => { value: Item[] | null };
```

Comment:
I see a lot of references to null, which in my opinion pollutes the code. Can we represent an empty selection with an empty array instead?

Reply:
Agree. Changed that.